### ISSUE TITLE
thoth: TypeError: Cannot call method 'on' of null

### ISSUE THREAD

**Issue by lloydde** [joyent/manta-thoth#121]
If a MANTA_USER without access is set then any `thoth` command will fail with `thoth: TypeError: Cannot call method 'on' of null`.

ENV

```
MANTA_USER=lloyd.dewolf
MANTA_KEY_ID=XXX_REMOVED_XXX
MANTA_URL=https://us-east.manta.joyent.com
THOTH_USER=thoth
```

My MANTA_USER is not a manta developer account, so I needed to also set MANTA_USER=thoth

ADDITIONAL DETAILS

When `THOTH_USER=thoth` was not set, then `thoth` is verses a "local install" and I don't receive this symptom. Hence, my belief it is an authentication/authorization issue.

```
$ thoth -h
thoth: using database at localhost:28015 (configured from source)
thoth: unrecognized command "-h"
Usage: thoth [command] [params]
...
```

**Comment by numericillustration** [comment]
I'm guessing this will be fixed by https://github.com/joyent/manta-thoth/pull/130  but in the mean time you can work around it by setting the ssh key fingerprint to an MD5 style one since this uses the older node-manta
`MANTA_KEY_ID=$(ssh-keygen -l -f ~/.ssh/id_rsa -E md5 | awk '{print $2}' | cut -d : -f 2-)`

**Comment by trentm** [comment]
Will be fixed by joyent/manta-thoth#144.

**Comment by trentm** [comment]
That fix is in. manta-thoth@1.0.0 is published with the fix.
